tests/e2e/helpers/decompressors/base-decompressors-populator.ts

```diff
@@ -0,0 +1,69 @@
+import { createClient } from 'redis';
+
+export abstract class BaseDatabasePopulator {
+    private client: ReturnType<typeof createClient>;
+
+    protected abstract createCompressedKeys(): Promise<void>;
+
+    constructor(private host: string, private port: string) {
+        const dbConf = { port: Number.parseInt(port), host, username: 'default' };
+        this.client = createClient(dbConf);
+
+        this.client.on('error', (error: string) => {
+            throw new Error(`Redis connection error: ${error}`);
+        });
+    }
+
+    public async populateDB(): Promise<void> {
```

Comment:
can you add comments with methods and parameters description for all these methods?

Reply:
fixed

Reply:
looks like you've added a comment only for this method